src/components/Article/index.tsx

```diff
@@ -48,6 +49,8 @@ export default class Article extends React.Component<ArticleProps> {
           case ContentType.Image:
             const image = JSON.parse(content.value) as ImageProps
             return <Image key={i} {...image} />
+          case ContentType.Break:
+            return <br />
```

Comment:
I think hr actually is what we want here – I always confuse the two https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr